plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -1145,20 +1055,19 @@ public boolean configure(final String name, final Map<String, Object> params) th
         }
 
         // Enable/disable IO driver for Qemu (in case it is not set CloudStack can also detect if its supported by qemu)
-        // Do not remove - switching it to AgentProperties.Property may require accepting null values for the properties default value
-        String enableIoUringConfig = (String) params.get(ENABLE_IO_URING_PROPERTY);
+        boolean enableIoUringConfig = AgentPropertiesFileHandler.getPropertyValue(AgentProperties.ENABLE_IO_URING);
```

Comment:
I have not tested it but in this case the default value is null, can we change the variable type to Boolean instead?